asttokens/util.py

```diff
@@ -218,6 +219,15 @@ def is_slice(node):
   )
 
 
+def is_empty_astroid_slice(node):
+  # type: (AstNode) -> bool
+  return (
+      node.__class__.__name__ == "Slice"
+      and not isinstance(node, ast.AST)
+      and node.lower is node.upper is node.step
```

Comment:
Should this say `is None` somewhere? Or are slices like `[5:5:5]` problematic in the same way as `[:]`?